GATK/4.1.3.0/BaseRecalibration.nf

```diff
@@ -0,0 +1,26 @@
+
+
+process BaseRecalibration {
+    tag {"BaseRecalibration ${sample_id}.${int_tag}"}
+    label 'GATK'

```

Comment:
Add version? This allows to use multiple GATK version in one workflow.

Reply:
This would indeed be better. This way you could have a generic config for every version you would like to run in the main config file.
```suggestion
    label 'GATK_4.1.3.0'
```